Resources/Audio/Weapons/licenses.txt

```diff
@@ -12,4 +12,6 @@ boxingbell.ogg taken from Herkules92 at https://freesound.org/people/Herkules92/
 
 block_metal1.ogg taken from https://github.com/Citadel-Station-13/Citadel-Station-13/commit/31c5996a5db8cce0cb431cb1dc20d99cac83f268 under CC BY-SA 3.0
 
-pierce.ogg taken from: https://github.com/tgstation/tgstation/commit/106cd26fc00851a51dd362f3131120318d848a53
\ No newline at end of file
+pierce.ogg taken from: https://github.com/tgstation/tgstation/commit/106cd26fc00851a51dd362f3131120318d848a53
+
+batonextend.ogg taken from https://github.com/tgstation/tgstation/commit/d4f678a1772007ff8d7eddd21cf7218c8e07bfc0
```

Comment:
make a rga for these (they are normally called attributes.yml or something, there are many examples of them in the repo)